amf-aml/shared/src/main/scala/amf/plugins/document/vocabularies/metamodel/domain/AnnotationMappingModel.scala

```diff
@@ -2,20 +2,26 @@ package amf.plugins.document.vocabularies.metamodel.domain
 
 import amf.core.metamodel.Field
 import amf.core.metamodel.Type.Iri
-import amf.core.metamodel.domain.{DomainElementModel, ModelDoc, ModelVocabularies}
+import amf.core.metamodel.domain.{DomainElementModel, ExternalModelVocabularies, ModelDoc, ModelVocabularies}
 import amf.core.vocabulary.{Namespace, ValueType}
 import amf.plugins.document.vocabularies.model.domain.AnnotationMapping
 
 object AnnotationMappingModel extends DomainElementModel with PropertyLikeMappingModel {
 
-  val Target: Field = Field(Iri,
-                            Namespace.Document + "target",
-                            ModelDoc(ModelVocabularies.AmlDoc,
-                                     "target",
-                                     "Target node IRI for which a specific annotation mapping can be applied"))
+  val Domain: Field = Field(
```

Comment:
Mmmm domain doesn't sound like the right way to define this restriction, at least for me. Also, I think we are using domain term with other meaning.
Is not specificaly your domain, but could be the domain of your target, in fact is the nodeTypeMapping property or the targetClass term

Reply:
Used `aml:domain` as a similar concept to `rdfs:domain` [https://www.w3.org/TR/rdf-schema/#h3_ch_domain](https://www.w3.org/TR/rdf-schema/#h3_ch_domain)

In RDFS an rdf:Property goes from a `rdfs:domain` to an `rdfs:range`. An `aml:domain` is the same concept but instead of at the schema level at the instance level. Schema level concepts belong to Vocabularies.

Reply:
But in this case the relation is de other way around. Not from the parent to the range, is a constraint of the property for the parent

Reply:
That is exactly my point. 
`Parent -(annotation mapping)-> Node Mapping` therefore 
```yaml
AnnotationMapping
  domain: Parent
  range: Node Mapping
```

Reply:
Target sounds ambiguous to me. Target can be interpreted as both domain and range

Reply:
@antoniogarrote what do you think about this?